Extensions/xkit_patches.js

```diff
@@ -1584,12 +1585,13 @@ XKit.tools.getParameterByName = function(name){
 
 				m_return.note_count = n_count;
 
-				m_return.avatar = $(obj).find(".post_avatar_image").attr('src');
+				m_return.avatar = post_data["tumblelog-data"].avatar_url;
 
 				m_return.tags = "";
-				if ($(obj).find(".post_tags").length > 0) {
+				var post_tags = $(obj).find(".post_tags") || $(obj).parents('.post')[1].find('.post_tags');
+				if (post_tags.length > 0) {
 					var to_return = "";
-					$(obj).find(".post_tags").find(".post_tag").each(function() {
```

Comment:
The logic you replace this line with loses the extra specificity of `.find(".post_tag")`, which causes the following:
Tags before this change: "reblog,ask,anon,ask,redacted-username"
Tags after this change: "reblog,ask,Ask redacted-username a question#anon,ask,redacted-username"

See https://www.tumblr.com/tagged/ask for example posts.